### PR TITLE
This should fix the IE11 issue and the issue when refreshing a survey.

### PR DESCRIPTION
**Pull request by wilblack**

### CODE REVIEW

File: src/viewpoint/services/vpApi-sync.js [hunk 1]
```diff
@@ -210,7 +210,7 @@ angular.module('vpApi.services')
             
             // Update status table
             var statusTable = $vpApi.db.getCollection('statusTable');
-            item = statusTable.find({'resourceId':resp.id});
+            var item = statusTable.find({'resourceId':resp.id});
```
Comment (wilblack) [inline src/viewpoint/services/vpApi-sync.js:213]: This should fix the IE11 saving issue.

File: src/viewpoint/services/survey-services.js [hunk 2]
```diff
@@ -329,7 +329,14 @@ angular.module('survey.services', [])
         }
 
         // Set the forEachItem
-        if (scope.current.formResp && typeof(scope.current.formResp.formForEachItem) !== 'undefined'){
+        var formForEachItem = null;
```
Comment (wilblack) [inline src/viewpoint/services/survey-services.js:332]: This should fix the survey not loading when coming back in issue.

**Comment by wilblack** [comment]
@dnseminara This branch should fix the foreach issues and the IE11 not saving issue. Can you test this on your local version. We should get this pushed up this morning.

**Comment by dnseminara** [comment]
closing PR as I can't merge it in via github - going manually

**Pull request closed without merging**